### PR TITLE
docs: add PyPI field and a few more projects

### PR DESCRIPTION
**Pull request by henryiii**
Adding a few more from #485, also a new `pypi:` field to make #605 easier.

### CODE REVIEW

File: docs/data/projects.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # name: required, printed name
 # gh: GitHub repo (eventually a url: could be added for non-github projects)
 # stars: GitHub repo (optional, if different from package, such as for Twisted)
+# pypi: The pypi name, if different from the github package name
```
Comment (henryiii) [inline docs/data/projects.yml:5]:
```suggestion
# pypi: The pypi name, if different from the GitHub package name
```

**Comment by henryiii** [comment]
We could in instead always use the pypi name as the "name".

**Comment by YannickJadoul** [comment]
You don't want PyPI links for all? Because e.g. for my Parselmouth, there's also a difference between "Parselmouth" and `praat-parselmouth`.

**Comment by joerick** [comment]
Seems like a good idea to me! I quite like keeping the separate name field, because some project pypi names are a little misleading, but I also wouldn't mind changing that if you prefer.

**Pull request merged**